tests/ci_tests/code_formatter/crewai/test_crewai_tasks.py

```diff
@@ -0,0 +1,200 @@
+"""Tests for coffee_maker/code_formatter/crewai/tasks.py"""
+
+from unittest import mock
+
+import pytest
+from crewai import Agent, Task
+
+from coffee_maker.code_formatter.crewai.tasks import (
+    EXPLANATIONS_DELIMITER_END,
+    EXPLANATIONS_DELIMITER_START,
```

Comment:
Line 6-10: Added a docstring to the function as specified in the style guide.
```suggestion
    """Creates a directory if it doesn't exist.\n\n    Args:\n        directory_path (str): The path to the directory.\n    """
```